legion/engine/physics/halfedgeface.hpp

```diff
@@ -23,6 +23,7 @@ namespace legion::physics
 		HalfEdgeEdge* startEdge;
 		math::vec3 normal;
 		math::vec3 centroid;
+        math::color DEBUG_color;
```

Comment:
This seems like a weird variable to save, I do understand that this is used to draw the face in the same color each time.